python/paddle/nn/layer/loss.py

```diff
@@ -119,26 +123,22 @@ def forward(self, input, label):
 
         if self.reduction not in ['sum', 'mean', 'none']:
             raise ValueError(
-                "The value of 'reduction' in cross_entropy_loss should be 'sum', 'mean' or 'none',"
-                " but received %s, which is not allowed." % self.reduction)
+                "The value of 'reduction' in cross_entropy_loss should be 'sum', 'mean' or"
+                " 'none', but received %s, which is not allowed." %
+                self.reduction)
 
         softmax_out = fluid.layers.softmax(input)
-        if self.weight is not None:
-            if isinstance(self.weight, fluid.framework.Variable):
-                softmax_out = fluid.layers.elementwise_pow(
-                    softmax_out, self.weight, axis=-1)
-            else:
-                raise ValueError(
-                    "The weight' is not a Variable, please convert to Variable.")
-
-        out = fluid.layers.cross_entropy(softmax_out, label)
+        log_softmax_out = fluid.layers.log(softmax_out)
```

Comment:
For LogSoftmax, it's better to reuse https://github.com/PaddlePaddle/Paddle/pull/23258 . Even through, that PR is also combiling `softmax` and `log`.   ( https://github.com/PaddlePaddle/Paddle/pull/23258#pullrequestreview-408609589 )

Reply:
Thanks, DONE